2/src/Pizza.ts

```diff
@@ -0,0 +1,28 @@
+import { Consumable } from "./Consumable";
+
+export class Pizza extends Consumable {
+    public readonly numberOfSlices: number;
+    private numberOfEatenSlices: number = 0;
+
+    constructor(value: number, weight: number, isSpoiled: boolean, numberOfSlices) {
```

Comment:
Please use object instead of 4 arguments. Also `numberOfSlices` does not have type. You can add TSLint rule not allow that